nautobot/ipam/filters.py

```diff
@@ -397,11 +392,24 @@ class IPAddressFilterSet(
         to_field_name="name",
         label="Namespace (name or ID)",
     )
+    has_interface_assignments = RelatedMembershipBooleanFilter(
+        field_name="interfaces",
+        method="_assigned_to_interface",
+        label="Has Interface Assignments",
+    )
 
     class Meta:
         model = IPAddress
         fields = ["id", "ip_version", "dns_name", "type", "tags", "mask_length"]
 
+    def _assigned_to_interface(self, queryset, name, value):
```

Comment:
Does this need to follow the `generate_query__...` pattern for reversibility by dynamic-groups?